src/view/context/content.hpp

```diff
@@ -31,10 +31,11 @@ namespace Context
 		void setCurrentlyPlaying(const lib::spt::track &track);
 
 		void setAlbum(const lib::spt::entity &albumEntity, const QPixmap &albumImage);
+		void setAlbumJpg(const lib::spt::entity &albumEntity, const std::string &data);
 
 	private:
 		/** Width and height of album */
-		static constexpr int albumSize = 64;
+		static constexpr int albumSize = 300;
```

Comment:
The small size should probably still be a fixed 64×64. The large one should be dynamic to whatever the width of the side panel is, and wouldn't need a fixed size like this.